WordPress/Sniffs/Theme/NoSanitizeCallBackSniff.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ * UseCapabilitiesNotRolesSniff
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ * WordPress_Sniffs_Theme_NoSanitizeCallbackSniff
+ *
+ * ERROR : Ensure that all calls the ->add_setting() for the Customizer include a
+ * sanitize_callback or a sanitize_js_callback parameter and that it's non-empty.
+ * An exception will need to be made for the (two) calls found in Kirki_Settings
+ * as they do correctly comply, but are wrappers for the real call.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @author   khacoder
+ */
+class WordPress_Sniffs_Theme_NoSanitizeCallBackSniff implements PHP_CodeSniffer_Sniff
+{
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		return array(
+			T_STRING,
+			T_CONSTANT_ENCAPSED_STRING,
+			T_VARIABLE,
+			T_LNUMBER,
+			T_SEMICOLON,
+		);
+	}//end register()
+
+	/**
+	 * Processes this test, when one of its tokens is encountered.
+	 *
+	 * @param PHP_CodeSniffer_File $phpcsFile The file being scanned.
+	 * @param int                  $stackPtr  The position of the current token
+	 *                                        in the stack passed in $tokens.
+	 *
+	 * @return void
+	 */
+	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
+
+		$tokens = $phpcsFile->getTokens();
+		$token  = $tokens[ $stackPtr ];
+		$types = array( T_STRING, T_CONSTANT_ENCAPSED_STRING, T_VARIABLE, T_LNUMBER, T_SEMICOLON );
+		if ( '$wp_customize' === $token['content'] ) {
```

Comment:
This is not guaranteed. The developer does not need to use `$wp_customize`. The variable could be named anything. This has become a bit of a standard as that is what core used.